bigbluebutton-config/bin/bbb-conf

```diff
@@ -1668,7 +1668,10 @@ if [ -n "$HOST" ]; then
     #
     # Update HTML5 client
     #
-    if [ -f $HTML5_DEFAULT_CONFIG ]; then
+    if [ -f $HTML5_ETC_CONFIG ]; then
+        yq w -i $HTML5_ETC_CONFIG public.kurento.wsUrl "wss://$HOST/bbb-webrtc-sfu"
+        yq w -i $HTML5_ETC_CONFIG public.pads.url      "$PROTOCOL://$HOST/pad"
+    else
```

Comment:
Good catch. I think `$HTML5_DEFAULT_CONFIG` should not be changed at all. Instead `$HTML5_ETC_CONFIG` should be created if it does not exist. Mentioning #14484 as there are probably more things to fix